ServerHttp/src/main/kotlin/io/raspberrywallet/server/Server.kt

```diff
@@ -20,10 +24,15 @@ import kotlinx.coroutines.experimental.launch
 
 
 class Server(private val manager: Manager) {
+    companion object {
+        const val PORT = 8080
+    }
+
     fun start() {
         val vertx = Vertx.vertx()
         vertx.deployVerticle(MainVerticle(manager)) {
-            if (it.succeeded()) println("Server started")
+            if (it.succeeded())
+                println("Server started at port $PORT")
```

Comment:
Can you use logger instead of println? Not sure If it makes sense in kotlin

Reply:
`println` is alias for `System.out.println`, so it is valid. But right, logger should be used instead